### PR TITLE
Reactive safety

### PR DESCRIPTION
**Pull request by Marco-Andarcia**
Modified safety controller to listen to the new bumpers topic.

### CODE REVIEW

**Comment by stonier** [comment]
We want to be careful here, this will probably break tutlebot.

@Marco-Andarcia an important point to keep in mind - this module should be a generic reactive safety controller and not actually know anything about a 'gopher' or a 'turtlebot' so we can minimise the amount of code we have to write.

File: yocs_safety_controller/src/library.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ bool SafetyController::init()
 
   enable_controller_subscriber_ = nh_priv_.subscribe("enable", 10, &SafetyController::enableCB, this);
   disable_controller_subscriber_ = nh_priv_.subscribe("disable", 10, &SafetyController::disableCB, this);
-  ranger_subscriber_ = nh_priv_.subscribe("rangers", 10, &SafetyController::rangerCB, this);
+  ranger_subscriber_ = nh_priv_.subscribe("/gopher/bumper", 10, &SafetyController::rangerCB, this);
```
Comment (stonier) [inline yocs_safety_controller/src/library.cpp:52]: Avoid hard coding locations like this as much as possible. Set them private (e.g. "rangers") and let launchers do the remapping.

**Comment by stonier** [comment]
@Marco-Andarcia let's review this early next week

**Comment by bit-pirate** [comment]
This is my bad. The changes are along the direction I initially pointed Marco at. Revised my thinking due too several issues, including the one breaking turtlebot (because of unreleased gopher_msgs).

Check out the bitbucket thread for our design discussion.

**Comment by stonier** [comment]
We're bringing in a new PR.

**Pull request closed without merging**